index.html

```diff
@@ -139,18 +139,22 @@ <h1 class="heading-about-me">About me</h1>
         <form class="input" action="https://formspree.io/f/mnqrljzr"
         method="POST">
           <div class="name-container">
-          <label for="firstname">
-            <input type="text" class="item1" id="firstname" name="firstname" maxlength="30" placeholder="First Name" required>
+            <label for="firstname">
+            <small id="small-1"></small>
+            <input type="text" class="item1" id="firstname" name="firstname" maxlength="30" placeholder="First Name">
```

Comment:
- [ ] Kindly add the `required` attribute to this input field to help with basic form validation